overrides/kubejs/server_scripts/ColdSweat.js

```diff
@@ -10,4 +10,93 @@ ServerEvents.recipes(event => {
         B: 'minecraft:bricks',
         C: 'minecraft:campfire'
     })
+    event.remove({ id: 'cold_sweat:thermometer'})
+    event.shaped('cold_sweat:thermometer', [
```

Comment:
thermometer changes i requested and then promptly did not set as req. changes lmoa